CONTRIBUTING.md

```diff
@@ -355,11 +356,9 @@ After that you can run your test using the standard `pytest` commands. For examp
 pytest -s -vv tests/path-to-tests/test_file.py
 ```
 
-### Running system tests
+### Running `system_tests` locally (internal-only)
 
-To run a test in [tests/system_tests](tests/system_tests),
-install [Docker Desktop](https://www.docker.com/products/docker-desktop/) and launch a local test
-server:
+If you're an internal engineer, launch a local test server:
```

Comment:
Add a sentence at the start of this section saying "Due to security limitations, external contributors cannot run system tests."